server/src/routes/api/v1/planetsRouter.js

```diff
@@ -20,8 +23,12 @@ planetsRouter.get("/:id", async (req, res) => {
   const id = req.params.id
   try {
     const planet = await Planet.query().findById(id)
+    const relatedReviews = await planet.$relatedQuery("reviews")
+    planet.reviews = relatedReviews
+    console.log(planet)
     return res.status(200).json({ planet: planet })
   } catch(error) {
+    console.log(error)
```

Comment:
remove all the console.logs, and you need to add serializers for planet and reviews